pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools>=72.2.0"]
```

Let's make it a bit more sparse:
```suggestion
requires = ["setuptools >= 72.2.0"]
```

Also, what are the differences with older versions? If they output comparable results (just not byte-to-byte), it might be fine to have a lower value here.

---

Yes, I believe it would be fine, but lack the experience and time to confirm that. I see no harm in aiming for a relatively high lower bound. There shouldn't be any conflicts due to build isolation, right?

---

For as long as it supports all required Python versions, this should be fine. The only case where conflicts may arise is downstream packaging, which you'll hear about if somebody won't be able to repackage it for Debian or something else with conservative software upgrades policies.

---

You were right, of course: #1193 got hit by this.